2-how-node-works/starter/modules.js

```diff
@@ -0,0 +1,18 @@
+// module.exports
+const C = require("./test-module-1");
+const calc = new C();
+console.log(calc.add(5, 5));
+console.log(calc.multiply(5, 2));
+console.log(calc.divide(10, 1));
+
+// exports
+const { add, multiply, divide } = require("./test-module-2"); // can pick imports
+console.log(add(1, 1));
+console.log(multiply(1, 2));
+console.log(divide(10, 5));
+
+// caching
+require(`./test-module-3`)();
+require(`./test-module-3`)();
+require(`./test-module-3`)();
+// loads the module only once and caches it...
```

Comment:
It seems wrong to add comments to the end of the code. Please try to use comments before the code